app/sequencescape/sequencescape/api/v2/request.rb

```diff
@@ -3,6 +3,8 @@
 class Sequencescape::Api::V2::Request < Sequencescape::Api::V2::Base
   FragmentSize = Struct.new(:from, :to)
 
+  has_one :submission
```

Comment:
where does this get used? thanks

Reply:
Setting up the relationship allows us to extract the submission id in LabwareCreators::WellFilter and the uuids when building the work orders.
This isn't entirely necessary when dealing with actual data coming across the API, as it will automatically configure the relationship based on the data, but it is important it we end up doing something like Sequencescape::Api::V2::request.new, such as when setting up data for tests.
Previously this slight difference in behaviour was hidden by the mocking that was going on.

Reply:
Ahh, actually it looks like this was one of the relationships I needed to keep mocking in for. I was having a few issues letting us both extract submission ids from the relationship, and access the actual object itself when we needed it. Works fine in actual code, but had issues setting up items in the right state.